vendor/assets/stylesheets/basscss/_border-colors.scss

```diff
@@ -11,6 +11,8 @@
 
 .border-gray   { border-color: $gray }
 
+.border-blue-lighter { border-color: $blue-lighter; }
```

Comment:
I think we should move this into our own `border.scss` vs the vendor stylesheet just in case we ever update Basscss.